sdk/appconfiguration/app-configuration/src/appConfigurationClient.ts

```diff
@@ -177,7 +177,7 @@ export class AppConfigurationClient {
       getGeneratedClientOptions(appConfigEndpoint, syncTokens, appConfigOptions)
     );
 
-    this.spanner = new Spanner<AppConfigurationClient>("Azure.Data.AppConfiguration", "appconfig");
+    this.spanner = new Spanner<AppConfigurationClient>("Azure.Data.AppConfiguration");
```

Comment:
Why are there code changes for tsconfig changes? Were these sneakily incorrect and are now caught by more strictness?

Reply:
(if so, awesome and please carry on!)

Reply:
Yesp yesp. This specific change shows the following error:

```
src/appConfigurationClient.ts:180:87 - error TS2554: Expected 1 arguments, but got 2.

180     this.spanner = new Spanner<AppConfigurationClient>("Azure.Data.AppConfiguration", "appconfig");
```